admin-ui/plugins/user-management/helper/validations.ts

```
@@ -0,0 +1,208 @@
+    ? Yup.string()
+        .nullable()
+        .transform((value) => (typeof value === 'string' && value.trim() === '' ? null : value))
+        .when('userPassword', {
+          is: (val: string | null | undefined) => {
+            if (val === null || val === undefined) return false
+            if (typeof val === 'string') {
+              return val.trim() !== ''
+            }
+            return true
+          },
+          then: (schema) =>
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)